BUILD

```diff
@@ -0,0 +1,8 @@
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+swift_library(
+    name = "Kronos",
+    module_name = "Kronos",
+    srcs = glob(["Sources/*.swift"]),
+    visibility = ["//visibility:public"]
```

Comment:
Not sure how many folks are using this but can you set:

```bzl
applicable_licenses = ["LICENSE"],
```

Reply:
How does this attribute work? Did not work for me out of the box

Reply:
wdym didn't work? I believe really it just holds this value for others to collect with something like https://github.com/bazelbuild/rules_license

Reply:
```terminal
ERROR: /Users/maxwellelliott/Development/Kronos/BUILD:3:14: in applicable_licenses attribute of swift_library rule //:Kronos: source file '//:LICENSE' is misplaced here (expected no files)
```

Reply:
hrm ok we can ignore for now